tensorflow/go/graph.go

```diff
@@ -133,7 +133,7 @@ func (g *Graph) ImportWithOptions(def []byte, options GraphImportOptions) error
 		return fmt.Errorf("unable to allocate memory")
 	}
 	defer C.free(buf.data)
-	C.memcpy(buf.data, unsafe.Pointer(&def[0]), buf.length)
+	C.memcpy(buf.data, C.CBytes(def), buf.length)
```

Comment:
I meant replace the C.malloc call above with C.CBytes.

```
// Go []byte slice to C array
// The C array is allocated in the C heap using malloc.
// It is the caller's responsibility to arrange for it to be
// freed, such as by calling C.free (be sure to include stdlib.h
// if C.free is needed).
func C.CBytes([]byte) unsafe.Pointer
```

Reply:
@mdaley change type of data.buf to C.CBytes?

Reply:
C.CBytes is a function, not a type.

Reply:
Ah, I see. Thanks.